sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -332,6 +332,9 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
     def apply(plan: LogicalPlan): Seq[SparkPlan] = plan match {
       case r: RunnableCommand => ExecutedCommand(r) :: Nil
 
+      case _: logical.ScriptTransformation =>
```

Comment:
@gatorsmile you can assume that only the `HiveSqlParser` supports this, and that a `HiveSqlParser` (and its builder) are only used within a `HiveContext`.